src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -121,26 +147,38 @@ function AttachmentCarouselPager({items, renderItem, initialIndex, onPageSelecte
         [],
     );
 
+    const Content = useMemo(
+        () =>
+            items.map((item, index) => (
+                <View
+                    key={item.source}
+                    style={styles.flex1}
+                >
+                    <CarouselItem
+                        // @ts-expect-error TODO: Remove this once AttachmentView (https://github.com/Expensify/App/issues/25150) is migrated to TypeScript.
+                        item={item}
+                        isSingleItem={items.length === 1}
+                        index={index}
+                        isFocused={index === activePageIndex && activeSource === item.source}
+                    />
+                </View>
+            )),
+        [activePageIndex, activeSource, items, styles.flex1],
```

Comment:
NAB: `styles.flex1` will never change so it doesn't make sense to be used with `useMemo()`